components/error.tsx

```diff
@@ -1,11 +1,22 @@
 import { H3, Panel } from '@bigcommerce/big-design';
 import { ErrorMessageProps } from '../types';
 
-const ErrorMessage = ({ error }: ErrorMessageProps) => (
-    <Panel>
-        <H3>Failed to load</H3>
-        {error && error.message}
-    </Panel>
-);
+const ErrorMessage = ({ error, renderPanel = true }: ErrorMessageProps) => {
+    if (renderPanel) {
+        return (
+            <Panel>
+                <H3>Failed to load</H3>
+                {error && error.message}
+            </Panel>
+        )
+    }
+
+    return (
+        <>
+            <H3>Failed to load</H3>
+            {error && error.message}
+        </>
+    )
+};
```

Comment:
it seems a bit redundant that we're repeating the same content (H3, error message) for both returns... we could probably refactor slightly:

```suggestion
const ErrorMessage = ({ error: { message }, renderPanel = true }: ErrorMessageProps) => {
    const errorContent = (
        <>
            <H3>Failed to load</H3>
            {message && message}
        </>
    );

    if (renderPanel) return <Panel>{errorContent}</Panel>;

    return errorContent;
};
```

Reply:
Good point, however I did it a little differently.